source/GameData.cpp

```diff
@@ -929,6 +931,23 @@ const string &GameData::Rating(const string &type, int level)
 
 
 
+// Strings for ship, bay type, and outfit categories.
+const vector<string> &GameData::Category(const string &type)
+{
+	return categories[type];
+}
+
+
+
+// Lists for categories as a set.
+const set<string> &GameData::CategorySet(const string &type)
+{
+	static const set<string> categorySet(categories[type].begin(), categories[type].end());
```

Comment:
This doesn't work how you want. If the first call is for ships, the call for outfits will also return the result from ships.

It's likely simpler to not have this `CategorySet` method at all.